### ISSUE TITLE
Get faster search for the site 

### ISSUE THREAD

**Issue by SrPatinhas** [DJScias/LoLES#11]
if we are on lolking, search on lolking...

**Comment by DJScias** [comment]
What do you mean by this? It's a bit cryptic.

**Comment by SrPatinhas** [comment]
No, we can see what is the links where we are, and we can add some html and css to the page to use the extension xD
For example 
![image](https://cloud.githubusercontent.com/assets/6379218/6353072/ece85d5a-bc3e-11e4-8b8e-993607ba55b9.png)

**Comment by DJScias** [comment]
Well, if you think you can figure out how then feel free to. I mean, it's a nice extra I suppose!
